tests/config/dapr_cosmosdb_query_state.yaml

```diff
@@ -34,5 +34,3 @@ spec:
     secretKeyRef:
       name: cosmosdb-secret
       key: queryCollection
-  - name: actorStateStore
```

Comment:
This was the root cause of increased failure rate of the reminder E2E tests.

Reply:
So, the root cause was that we were using cosmosdb with actorStateStore = true? In that case, why were the failures intermittent? Should we not have had a 100% failure rate with this kind of misconfiguration?

Reply:
I think I understand, based on an offline conversation, the actual issue is that we had two actoreStateStores configured and we were racing between which one to use?

Reply:
Correct.